test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -925,7 +926,9 @@ def revert_mom(self, mom, force=False):
         name = mom.shortname
         if mom.platform == 'cray' or mom.platform == 'craysim':
             try:
-                self.server.manager(MGR_CMD_DELETE, NODE, None, '')
+                if self.del_all_nodes:
```

Comment:
@hirenvadalia , basically this is for the multimom execution where user specifies multiple moms through -p option so in that case it should not delete all the moms on each revet_mom call but it should do it only first time which can be handled by the new flag which I defined del_all_nodes. One thing I did wrong here is that I mentioned the flag at class level, which I think I need to put under "def setUp(self)".